bin/run

```diff
@@ -4,34 +4,59 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+const semver = require('semver');
+const chalk = require('chalk');
```

Comment:
Moving the new dependencies to `checkNodeVersion` method may make this more readable (see next comment suggestion): 
``` js
function checkNodeVersion() {
  const semver = require('semver');
  const chalk = require('chalk');
  //...
```